src/collections/_documentation/development/sdk-dev/event-payloads/contexts.md

```diff
@@ -3,9 +3,9 @@ title: Contexts Interface
 sidebar_order: 9
 ---
 
-The context interfaces provide additional context data. Typically, this is data
```

Comment:
There was a typo here, `context interfaces` => `Contexts Interface`.
The rest is just rewrapping the paragraph.

Reply:
It was actually meant as plural since there are multiple contexts, but you're right. Singular is more consistent, since it is actually just one interface.